eks-anywhere-common/Addons/Partner/Tetrate.io/namespace.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: istio-system
+  labels:
+    aws.conformance.vendor: tetrate.io
+    aws.conformance.vendor-solution: tetrate-istio-distribution
```

Comment:
Could you please add additional label ` aws.conformance.vendor-solution-version: X.X.X`. This is something we added recently.

Reply:
updated